package.json

```diff
@@ -104,7 +104,7 @@
   "eslintConfig": {
     "root": true,
     "parserOptions": {
-      "ecmaVersion": 5
+      "ecmaVersion": 6
```

Comment:
also configurations are merged, so only the `parserOptions` property would need to be set.